runtime/flang/CMakeLists.txt

```
@@ -560,12 +560,21 @@ set_property(
  PG_PIC
  )
 
-set_source_files_properties(
-  ${I8_FILES_DIR}/red_norm2_stride1.c
-  red_norm2_stride1.c
-  PROPERTIES
-  COMPILE_FLAGS "-ffast-math"
-  )
+if (NOT MSVC)
```

To support using clang.exe which has GNU like command line flags.
```suggestion
if (NOT MSVC OR "${CMAKE_C_COMPILER_FRONTEND_VARIANT}" STREQUAL "GNU")
```

---

Yes, you're right. Thanks.